src/Workspaces/Core/Portable/Diagnostics/DiagnosticDataSerializer.cs

```diff
@@ -49,7 +49,7 @@ public async Task<bool> SerializeAsync(IPersistentStorageService persistentServi
                 WriteDiagnosticData(writer, items, cancellationToken);
             }
 
-            using var storage = await persistentService.GetStorageAsync(project.Solution, cancellationToken).ConfigureAwait(false);
+            await using var storage = await persistentService.GetStorageAsync(project.Solution, cancellationToken).ConfigureAwait(false);
```

Comment:
❔ How does `await using` interact with `ConfigureAwait`?

Reply:
it seems to be respected: https://devblogs.microsoft.com/dotnet/configureawait-faq/#can-i-use-configureawait-when-await-using-an-iasyncdisposable

Reply:
Ah, it is respected, but only if configured. Currently we have the equivalent of omitting the `ConfigureAwait` call.

Reply:
ok.  have made this change :)